src/database/queries/bulkUploads/updateStatusById.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE bulk_uploads SET
+  status = :status
+WHERE id = :id
+RETURNING
+  id as "id",
+  file_name as "fileName",
+  source_key as "sourceKey",
+  status as "status",
+  created_at AS "createdAt"
+
```

Comment:
Right, although isn't there some value in having this be a common pattern (you can always expect the affected entity to be returned when querying) -- similar to how REST will return the updated entity on a `PUT`?

That can always be added at the time it's needed, though our type safety is *weakest* at the point of database interaction since we're manually typing all DB responses (which is to say, inconsistency in return pattern is more likely to lead to odd behavior since you can't just assume return type based on the query type any longer and have to remember to inspect the query whenever you use it to see what it returns).